.github/workflows/build_and_test.yml

```diff
@@ -938,7 +938,7 @@ jobs:
     container:
       image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/base:pinned
       options: --init
-    needs: [ promote-images, tag ]
+    needs: [ tag ]
```

Comment:
If we run it right after the `tag`, how will the job in `build-custom-extensions` get images for the default extension?

Reply:
not sure what you mean. `build-custom-extensions` does not pull any images defined in this repo
it just clones `neondatabase/postgres` and builds extensions on top of that

Reply:
which is why CI can pass for this the remote repo in this PR

Reply:
> it just clones neondatabase/postgres and builds extensions on top of that
if this is wrong, lmk

Reply:
From https://github.com/neondatabase/neon/pull/5085, I understand that `build-custom-extensions` uploads default extensions. 
Which job builds default extensions?

Reply:
`build-custom-extensions` builds and uploads to S3 all custom extensions (both "public" and "private"). the point of #5085 is to make this all happen in one place rather than splitting over 2 separate repos. 

`neon` repo Dockerfile is still responsible for building extensions that are baked into the compute image. For now, we are not (afaik) planning to immediately move all extensions from compute image to remote storage. 

So some the extensions in the compute image are built in neon and the remote extensions that go to S3 are built and uploaded in the build-custom-extensions repo.

Reply:
fair enough

Reply:
on the other hand, custom extensions is probably the main use case for now, 
so I will hold off on renaming that repo